routes/api/list.js

```diff
@@ -89,6 +89,45 @@ exports = module.exports = function(req, res) {
 
 
 		break;
+		
+		case 'getAll':
```

Comment:
I think a dedicated router at `/api/v2/lists/` would be better than a `getAll` case here.

You may also want to check out [Chapter 8. Building RESTful APIs from Programming JavaScript Applications](http://chimera.labs.oreilly.com/books/1234000000262/ch08.html) for ideas.